### ISSUE TITLE
Quarto 1.4.303 triggered Shinylive / deno.land error

### ISSUE THREAD

**Issue by garrettgman** [quarto-dev/quarto#242]
I updated my Macbook to the developer version of quarto, 1.4.303, and noticed that I could no longer render quarto documents that contained Shinylive apps, as I reported [here](https://github.com/quarto-ext/shinylive/issues/15#issuecomment-1675375077).

Rolling back to quarto 1.3.450 made the error go away.

While I was using 1.4.303, I could create the error by opening a new quarto doc and pasting in the example shinylive contents below [from the shinylive repo](https://github.com/quarto-ext/shinylive#usage).

````
---
title: Shinylive in Quarto example
format: html
filters:
  - shinylive
---

This is a Shinylive application embedded in a Quarto doc.

```{shinylive-python}
#| standalone: true

from shiny import *

app_ui = ui.page_fluid(
    ui.input_slider("n", "N", 0, 100, 40),
    ui.output_text_verbatim("txt"),
)

def server(input, output, session):
    @output
    @render.text
    def txt():
        return f"The value of n*2 is {input.n() * 2}"

app = App(app_ui, server)
```
````

Running `quarto preview` returned
```
error: Uncaught SyntaxError: The requested module 'https://deno.land/std/io/buffer.ts' does not provide an export named 'readLines'
import { readLines } from "https://deno.land/std/io/buffer.ts";
         ^
    at <anonymous> (file:///Users/garrettgrolemund/Library/Caches/shinylive/shinylive-0.1.5/scripts/codeblock-to-json.js:2776:10)
Error running filter /Applications/quarto/share/filters/main.lua:
...
```

**Comment by jjallaire** [comment]
This is going to require a change in the `shinylive` extension to work with the updated version of the deno standard library included with v1.4 (unfortunately it is still shifting around a bit). Could you re-post the issue there?

Note that if `shinylive` wants to make 2 versions available (one for use with v1.3 and one for v1.4) they can use a branch/tag based installation (as described here: https://quarto.org/docs/extensions/distributing.html#github-distribution) and just update docs accordingly to help users pick the right one.

**Comment by garrettgman** [comment]
Thank you @jjallaire. I reposted the issue at <https://github.com/quarto-ext/shinylive/issues/18> and will close this one.
